.github/workflows/coverage.yml

```diff
@@ -23,9 +22,9 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Test with coverage
-      run: go test -gcflags=-l -coverprofile=coverage.txt -covermode=atomic ./...
+      run: go test -v -race -gcflags=-l -coverprofile=coverage.txt -covermode=atomic ./...
 
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./coverage.txt
+    - name: Upload coverage report
+      uses: codecov/codecov-action@v4
+      env:
+        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
What is this token? Do we need to set it?

Reply:
Codecov token, you can get in codecov configuration:

https://app.codecov.io/gh/pactus-project/pactus/